### PR TITLE
Revert iam_additive behaviour

### PR DESCRIPTION
**Pull request by ludoo**
The `iam_additive` variable behaviour has been reverted so it's keyed by roles, to make it more resilient with dynamic values. A new `iam_additive_members` has been added to preserve the previous use case.

### CODE REVIEW

File: modules/organization/main.tf [hunk 1]
```diff
@@ -16,13 +16,17 @@
...
 locals {
   iam_additive_pairs = flatten([
-    for member, roles in var.iam_additive : [
-      for role in roles :
-      { role = role, member = member }
+    for role, members in var.iam_additive : [
+      for member in members : { role = role, member = member }
+    ]
+  ])
+  iam_additive_member_pairs = flatten([
```
Comment (juliocc) [inline modules/organization/main.tf:23]: isn't [transpose()](https://www.terraform.io/docs/configuration/functions/transpose.html) what you want?

File: modules/project/main.tf [hunk 2]
```diff
@@ -16,13 +16,17 @@
...
 locals {
   iam_additive_pairs = flatten([
-    for member, roles in var.iam_additive : [
-      for role in roles :
-      { role = role, member = member }
+    for role, members in var.iam_additive : [
+      for member in members : { role = role, member = member }
+    ]
+  ])
+  iam_additive_member_pairs = flatten([
```
Comment (juliocc) [inline modules/project/main.tf:23]: idem

**Review by juliocc — COMMENTED** [review]
PTAL at comments but LGTM.

**Comment by ludoo** [comment]
As discussed offline ;)

**Pull request merged**